### PR TITLE
feat(brew): add `--prefix` subcommand

### PR DESCRIPTION
**Pull request by bengiv**
Adding the --prefix subcommand

### CODE REVIEW

File: src/brew.ts [hunk 1]
```diff
@@ -1203,6 +1203,15 @@ const completionSpec: Fig.Spec = {
...
     },
+    {
+      name: "--prefix",
+      description: "Prefix of <formula>",
+      args: {
+        isVariadic: true,
+        name: "formula",
+        generators: formulaeGenerator,
+      },
+    },
```
Comment (mschrage) [inline src/brew.ts:1214]: This looks like it should be an option rather than a subcommand?

Comment (ben5vik) [reply src/brew.ts:1214]: Hmm

Comment (ben5vik) [reply src/brew.ts:1214]: Well we know it wasn't in autocompletion and this adds it 😅

Comment (SeparateRecords) [reply src/brew.ts:1214]: It looks like an option but @ben5vik is right, it is actually a subcommand 🤨 even has its own non-global options!

@ben5vik: Can you add `type: "option"` and `icon: "fig://icon?type=option` to this, just to make it look more like an option in the list?

Comment (SeparateRecords) [reply src/brew.ts:1214]: Ah, and also a comment explaining that _yes_ it looks like a subcommand but _no_ it actually isn't one

Comment (SeparateRecords) [inline src/brew.ts:1214]:
```suggestion
    {
      // This looks like an option, but it's actually a subcommand with its own options
      // See: brew help --prefix
      name: "--prefix",
      description: "Prefix of <formula>",
      icon: "fig://icon?type=option",
      args: {
        isVariadic: true,
        name: "formula",
        generators: formulaeGenerator,
      },
      options: [
        {
          name: "--unbrewed",
          description: "List files in Homebrew's prefix not installed by Homebrew",
        },
        {
          name: "--installed",
          description:
            "Outputs nothing and returns a failing status code if formula is not installed"
        },
      ],
    },
```

**Comment by fedeci** [comment]
recheck

File: src/brew.ts (hunk 1 above)
Comment (fedeci) [inline src/brew.ts:1214]:
```suggestion
    {
      // NOTE: this is actually a command even if it has the double dash in the front
      name: "--prefix",
      description: "Prefix of <formula>",
      type: "option",
      args: {
        isVariadic: true,
        name: "formula",
        generators: formulaeGenerator,
        },
      options: [
        {
          name: "--unbrewed",
          description: "List files in Homebrew's prefix not installed by Homebrew",
        },
        {
          name: "--installed",
          description:
            "Outputs nothing and returns a failing status code if formula is not installed",
        },
      ],
    },
```

**Pull request closed without merging**